packages/viewer/src/lib/interactions/__tests__/interactionApiOrthographic.spec.ts

```diff
@@ -155,7 +155,7 @@ describe(InteractionApiOrthographic, () => {
       expect(update.mock.calls[1][0].lookAt).toMatchObject({
         x: -1,
         y: 0,
-        z: 0,
+        z: 24.999999999999996,
```

Comment:
https://jestjs.io/docs/expect#expectclosetonumber-numdigits

```suggestion
        z: expect.closeTo(25, 5),
```

Reply:
Good idea, updated!